nomos-services/blend/src/lib.rs

```diff
@@ -208,6 +205,16 @@ where
     }
 }
 
+impl<Backend, Network, RuntimeServiceId> Drop for BlendService<Backend, Network, RuntimeServiceId>
+where
+    Backend: BlendBackend<RuntimeServiceId> + 'static,
+    Network: NetworkAdapter<RuntimeServiceId>,
+{
+    fn drop(&mut self) {
+        self.backend.shutdown();
+    }
+}
+
```

Comment:
Iirc once the abort handle is droped (which will occur after the older run scope) it just aborts the launched tasks. But I think this is even better as it is explicit and easier to read.

Reply:
I think its better to keep the current explicit impl with the manual `Drop`. But yeah, I think you may also just remove the `shutdown` function.